edit.html

```diff
@@ -1,192 +1,19 @@
 <!doctype html>
 <html>
-
 <head>
   <title>xrpackage | edit</title>
   <link href="index.css" rel=stylesheet>
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Muli:wght@400;600&display=swap" rel="stylesheet">
   <script src="https://kit.fontawesome.com/0735724151.js" crossorigin="anonymous"></script>
-  <!-- <script src="https://rawcdn.githack.com/ethereum/web3.js/a6ddec59e65116853435f203b25cb9c55824d084/dist/web3.min.js"></script> -->
+  <link href="./components/styles/index.css" rel=stylesheet>
```

Comment:
If this is used for components, they should have their own tags for it. Otherwise this doesn't work for 3D.